src/NodaTime/CalendarSystem.cs

```diff
@@ -602,6 +603,9 @@ internal Era GetEra([Trusted] int absoluteYear)
         /// </summary>
         /// <param name="yearMonthDay">The value to validate.</param>
         [Conditional("DEBUG")]
+#if NET45
```

Comment:
and `!DEBUG`?

Reply:
No need - we don't run the coverage tests under debug, and if we did (and ran not under net45) it would hit anyway :)